pkg/internal/constants.go

```diff
@@ -1,8 +1,11 @@
 package internal
 
 const (
-	ConfigNamespace        = "openshift-config"
-	ConfigManagedNamespace = "openshift-config-managed"
-	InstallerConfigMap     = "openshift-install"
-	ManifestsConfigMap     = "openshift-install-manifests"
+	ClusterVersionNamespace = "openshift-cluster-version"
```

Comment:
we don't need this now that the ConfigMaps have moved.

Reply:
/hold to address this comment

Reply:
removed